src/DynamoApplications/StartupUtils.cs

```diff
@@ -377,7 +377,10 @@ private static bool PreloadASM(string asmPath, out string geometryFactoryPath, o
                 HostAnalyticsInfo = info,
                 CLIMode = CLImode
             };
-            config.AuthProvider = CLImode ? null : new Core.IDSDKManager();
+
+            // ALIAS does needs a total network traffic blackout.
+            config.AuthProvider = null;
```

Comment:
Would this cause crash, if you click on login in Dynamo?

Reply:
No crash. The login button is still visible. Clicking it does nothing.